src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
+    : people.filter((p) => p.sex === 'f');
```

Comment:
```suggestion
    ? people.filter(({ name, sex }) => (
        sex === 'f' && people.find(({ mother }) => mother === name)))
    : people.filter(({ sex }) => sex === 'f');
```
use destructuring and if you move your shortened return statement in arrow function to a newline, use round brackets